index.html

```diff
@@ -208,6 +208,19 @@ <h4 class="social-links">LET’S CONNECT</h4>
       </ul>
     </div>
   </section>
+  <section class="contact">
+    <div class="form-title">
+      <h1>Contact me</h1>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+      <form class="form" action="https://formspree.io/f/meqwkove" method="post">
+        <input type="text" name="first_name" maxlength="30" required placeholder="  Your Name">
+        <input type="email" name="email" id="email" required placeholder="  Your Email Address">
+        <textarea name="message" id="text" maxlength="500" required placeholder="Write your message here"></textarea>
+        <button type="submit">Get in touch</button>
```

Comment:
[ OPTIONAL ] Kindly do your best to replicate figma design as this button has a different background than in figma file 
**figma :**
![get in touch white background](https://user-images.githubusercontent.com/59707859/221956440-638d38ac-9446-474d-b450-714e9c82c152.PNG)

Reply:
**Hi @ souad988 ,**

Thanks for your time and for reviewing my code.  I have altered the button background color according to the Figma design.I do not know how I did not notice it. 

However many thanks for the approval and put this change optional. 🙏